README.md

```diff
@@ -49,7 +49,7 @@ You can see a (customized) instance of Strapping in action by visiting http://oV
 2. Clone the repository:
 
    ```
-   git clone https://github.com/OSAS/strapping-mediawiki strapping
+   git clone https://github.com/axxie/strapping-mediawiki strapping
```

Comment:
axxie:
> I definitely don't want to change that line in your branch.

Btw not "my" branch. I am just a user / github 'watcher' of this project.